api/tests/unit/domain/models/PixEduBadgeAcquisitionOrderer_test.js

```diff
@@ -80,18 +80,18 @@ describe('Unit | Domain | Models | PixEduBadgeAcquisitionOrderer', function () {
 
             // then
             expect(highestBadge).to.deep.equal(
-              domainBuilder.buildBadgeAcquisition.forPixEduFormationContinue1erDegreInitie()
+              domainBuilder.buildBadgeAcquisition.forPixEduFormationContinue2ndDegreAvance()
             );
           });
         });
 
-        context('when there is no PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_INITIE badge acquisition', function () {
-          context('when there is a PIX_EDU_FORMATION_INITIALE_1ER_DEGRE_INITIE badge acquisition', function () {
-            it('should return the PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_EXPERT badge acquisition', function () {
+        context('when there is no PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_AVANCE badge acquisition', function () {
+          context('when there is a PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_AVANCE badge acquisition', function () {
+            it('should return the PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_FORMATEUR badge acquisition', function () {
```

Comment:
Pareil, ca devrait etre 
```suggestion
            it('should return the PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_AVANCE badge acquisition', function () {
```

Reply:
Corrigé